cli/src/cli/index.ts

```diff
@@ -152,7 +161,16 @@ export const runCli = async () => {
     if (cliResults.flags.trpc) cliResults.packages.push("trpc");
     if (cliResults.flags.tailwind) cliResults.packages.push("tailwind");
     if (cliResults.flags.prisma) cliResults.packages.push("prisma");
+    if (cliResults.flags.drizzle) cliResults.packages.push("drizzle");
     if (cliResults.flags.nextAuth) cliResults.packages.push("nextAuth");
+
+    if (cliResults.flags.prisma && cliResults.flags.drizzle) {
+      console.warn(
+        "Incompatible combination Prisma + Drizzle. Falling back to Prisma only.",
```

Comment:
this is just for the CI flags

when you're using the CLI it will keep prompting you if you add invalid stuff (see `validate` in the `promptPackages`:

![image](https://github.com/t3-oss/create-t3-app/assets/51714798/cc787aa9-e283-4baa-8202-57130501a6a8)

Reply:
Wouldn't a separate question be better? If the default settings are what most users use, it's comfortable to move forward without thinking about the options.